script/godep

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
Do you use godep for managing your dependencies? If not, you can get rid of this file.